life_expectancy/cleaning.py

```diff
@@ -1,44 +1,78 @@
 import argparse
+from pathlib import Path
 import pandas as pd
 
-parser = argparse.ArgumentParser(description='filters data by region.')
-parser.add_argument('--region', action='store', default='PT')
-args, _ = parser.parse_known_args()
 
-def clean_data(region=args.region):
+def load_data() -> pd.DataFrame:
     """
-    This function loads and cleans the data from eu_life_expectancy_raw dataset
-    and saves the cleaned dataset on the data folder.
+    This function loads the data eu_life_expectancy
+    Returns:
+        df: eu_life_expectancy as Pandas dataframe 
+    """
+    path = Path.cwd() / './life_expectancy/data/eu_life_expectancy_raw.tsv'
```

Comment:
Be careful with this... this path is relative to the command line, meaning that if you don't run this command at the exact spot, it will fail. A more robust alternative is to set the path relative to the current file:

```python
path = Path(__file__).parent / 'data' / 'eu_life_expectancy_raw.tsv'
```